EpikCord/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class NotFound:
```

Comment:
All exception from the lib should inherit from a base exception to make it clear that it is bounded to the lib.

Having a exception tree allow the  user to have more control in regard to the exception that can be prevented, wihout having to set them all manually

Reply:
Agreed.